test/models/developer_test.rb

```diff
@@ -213,7 +213,7 @@ class DeveloperTest < ActiveSupport::TestCase
     assert developer.featured?
     assert_includes Developer.featured, developer
 
-    travel 2.hours
+    travel 3.hours
```

Comment:
@joemasilotti not sure if this is the best solution here but this test was failing for me locally and in CI. I didn't change anything that would have affected it and when I reverted back to a commit that had been passing it also failed there, so I assume it has something to do with the time of day this is being run? I increased it an extra hour and now it passes.

Reply:
I'm assuming we are right on the edge case of an hour break in some way. Increasing the time to 3 hours is fine by me. I don't think it is worth diving into any more. Thanks for the quick fix!